phpunit/html/WP_HTML_Walker_Test.php

```diff
@@ -686,123 +757,123 @@ public function data_malformed_tag() {
 				'<hr id=code>><span>test</span>',
 				'<hr foo="bar" class="firstTag" id=code>><span class="secondTag">test</span>',
 			),
-			'1' => array(
+			'1'                                            => array(
```

Comment:
please no haha - was this auto-formatted?
it's going to be such a mess to keep these aligned

looks like we can also do this

```php
yield '1' => array(
   …
);
```

Reply:
I ran phpcbf on this file, I think the auto checks won’t pass otherwise

Reply:
We can add it as an entry in the array, plus we can pass it to the assertion as the 3rd param, so it hints at the failed case. We solve the formatting issue that is a huge limitation here and we get better failure messages.